test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -53,8 +53,12 @@ def ec2_pytorch_inference(image_uri, processor, ec2_connection, region):
     repo_name, image_tag = image_uri.split("/")[-1].split(":")
     container_name = f"{repo_name}-{image_tag}-ec2"
     model_name = "pytorch-densenet"
+    if processor is "eia":
+        if "1.3.1" in image_tag:
```

Comment:
Use https://github.com/aws/deep-learning-containers/blob/048d28b15d6b87796067428296668b5a8b8d5738/test/test_utils/__init__.py#L655-L676 to find the framework version of the image, rather than searching for the exact string in the `image_tag`.

This will allow easier unit testing, and will also allow us to improve the implementation that finds the framework version on an image URI everywhere in the code, if needed.